packages/shared/prisma/migrations/20240917072611_remove_covered_indexes/migration.sql

```diff
@@ -0,0 +1,18 @@
+-- DropIndex
+-- Removes indexes that are covered by other indexes on the same table, e.g. (project_id) if (project_id, timestamp) exists.
+DROP INDEX CONCURRENTLY IF EXISTS "traces_project_id_idx"; -- covered by traces_project_id_timestamp_idx
+DROP INDEX CONCURRENTLY IF EXISTS "datasets_project_id_idx"; -- covered by datasets_project_id_name_key (unique)
```

Comment:
This is an interesting case, because datasets_project_id_idx is a hash index while datasets_project_id_name is a standard btree. This may make for slightly faster reads using the first index so also happy to revert this line if there is a strong preference for it.

Reply:
Agree, i would delete this.